MIGRATING.md

```diff
@@ -1,6 +1,6 @@
 # Migrate from the SignalFx Tracing Library for Ruby
 
-The SignalFx Tracing Library for Ruby is deprecated. Replace it with the
+The SignalFx Tracing Library for Ruby will soon be deprecated. Replace it with the
```

Comment:
I think this is too vague. Do we have any dates for this? Can we say something like "will be deprecated by the end of 2022", for instance?